From 076aff299040287557d2a7a35140edf368d1f8f3 Mon Sep 17 00:00:00 2001
From: Olivier Fourdan <olivier.fourdan@intel.com>
Date: Fri, 19 Apr 2013 10:02:45 +0200
Subject: build: Add path to x86_64 toolchain

Bug: ADEV-116
Category: Device Enablement
Domain: AOSP-build
Severity: High
Origin: Internal
Upstream-candidate: Yes

Change-Id: Id139d3264fed7bfacf32ba9cc28e1fff75e166e1
Signed-off-by: Olivier Fourdan <olivier.fourdan@intel.com>
---
 core/dumpvar.mk |    6 ++++++
 1 files changed, 6 insertions(+), 0 deletions(-)

diff --git a/core/dumpvar.mk b/core/dumpvar.mk
index ef601e6..c52c1bb 100644
--- a/core/dumpvar.mk
+++ b/core/dumpvar.mk
@@ -26,6 +26,12 @@ else ifeq ($(TARGET_ARCH),x86)
         # this should be copied to HOST_OUT_EXECUTABLES instead
         ABP:=$(ABP):$(PWD)/prebuilts/gcc/$(HOST_PREBUILT_TAG)/x86/i686-linux-android-$(TARGET_GCC_VERSION)/bin
     endif
+
+# Add the x86_64 toolchain bin dir if it actually exists
+    ifneq ($(wildcard $(PWD)/prebuilts/gcc/$(HOST_PREBUILT_TAG)/x86/x86_64-linux-android-$(TARGET_GCC_VERSION)/bin),)
+        # this should be copied to HOST_OUT_EXECUTABLES instead
+        ABP:=$(ABP):$(PWD)/prebuilts/gcc/$(HOST_PREBUILT_TAG)/x86/x86_64-linux-android-$(TARGET_GCC_VERSION)/bin
+    endif
 endif
 
 # Add the mips toolchain bin dir if it actually exists
-- 
1.7.4.1

